scripts/chrome/jumpcloud/mac/v1/com.google.Chrome.mobileconfig

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>PayloadContent</key>
+	<array>
+		<dict>
+			<key>ExtensionSettings</key>
+			<dict>
+				<key>cmobkdiplndgpjodaioofofmcikimbdb</key>
+				<dict>
+					<key>installation_mode</key>
+					<string>force_installed</string>
+					<key>update_url</key>
+					<string>https://clients2.google.com/service/update2/crx</string>
+				</dict>
+			</dict>
+			<key>PayloadDescription</key>
+			<string>com.google.Chrome</string>
+			<key>PayloadDisplayName</key>
+			<string>com.google.Chrome</string>
+			<key>PayloadIdentifier</key>
+			<string>io.se.kandji.B1F4414B-E6EC-43BA-8894-8885693FE856</string>
+			<key>PayloadOrganization</key>
+			<string>Kandji, Inc.</string>
+			<key>PayloadType</key>
+			<string>com.google.Chrome</string>
+			<key>PayloadUUID</key>
+			<string>6738811D-67CA-4425-B92E-BB9AA2352065</string>
+			<key>PayloadVersion</key>
+			<integer>1</integer>
+		</dict>
+	</array>
+	<key>PayloadDescription</key>
+	<string>com.google.Chrome</string>
+	<key>PayloadDisplayName</key>
+	<string>com.google.Chrome</string>
+	<key>PayloadIdentifier</key>
+	<string>io.kandji.customprofile.7D0F6DE9-68E9-409A-AD5F-CF183CA976B3</string>
```

Comment:
Is this some random ID or something of us ?

Reply:
Randomly generate while converting plist to mobile config
ref: https://github.com/kandji-inc/support/tree/main/Plist2MobileConfig